Authentication/FaceDetection.py

```diff
@@ -83,15 +83,16 @@ def detectFace():
             if matches[matchIndex]:
                 name = classNames[matchIndex]
                 # * Following commands are not important but will create a square around the detected face and the name under it.
-                # y1,x2,y2,x1 = faceLoc
-                # y1, x2, y2, x1 = y1*4,x2*4,y2*4,x1*4
-                # cv2.rectangle(img,(x1,y1),(x2,y2),(0,255,0),2)
-                # cv2.rectangle(img,(x1,y2-35),(x2,y2),(0,255,0),cv2.FILLED)
-                # cv2.putText(img,name,(x1+6,y2-6),cv2.FONT_HERSHEY_COMPLEX,1,(255,255,255),2)
+                y1,x2,y2,x1 = faceLoc
+                y1, x2, y2, x1 = y1*4,x2*4,y2*4,x1*4
+                cv2.rectangle(img,(x1,y1),(x2,y2),(0,255,0),2)
+                cv2.rectangle(img,(x1,y2-35),(x2,y2),(0,255,0),cv2.FILLED)
+                cv2.putText(img,name,(x1+6,y2-6),cv2.FONT_HERSHEY_COMPLEX,1,(255,255,255),2)
```

Comment:
Well apparently there is no need for the camera to draw a box or to be displayed on the screen but for now I m merging the commit..